### ISSUE TITLE
nonce & accountid

### ISSUE THREAD

**Issue by w5b** [empdarkness/warframe-irc#1]
hey, in the recent patched warframe fixed the ability to get the nonce and accountid through the store. is there any other way to do so?

**Comment by empdarkness** [comment]
I believe both should be able to be pulled out of memory. Like for example, starting a recipe in foundry sends a request to `/api/startRecipe.php?accountId={accountId}&nonce={nonce}&ct=MSI` with the respective account id and nonce of your current login.
Probably not an easier way if the store is patched.

**Comment by w5b** [comment]
Yeah I already know how to get it from memory, but I hoped there would be an easier way. Thanks for the help tho
